changelog/updates/2023-05-03-weekly-updates.md

```diff
@@ -0,0 +1,7 @@
+- sqlite ([3.41.2](https://sqlite.org/releaselog/3_41_2.html))
+- GLib ([2.74.6](https://gitlab.gnome.org/GNOME/glib/-/tags/2.74.6))
+- libxml2 ([2.10.4](https://gitlab.gnome.org/GNOME/libxml2/-/tags/v2.10.4))
+- bpftool ([6.2.1](https://kernelnewbies.org/LinuxChanges#Linux_6.2.Tracing.2C_perf_and_BPF))
+- curl ([8.0.1](https://curl.se/changes.html#8_0_1))
+- e2fsprogs ([1.47.0](https://e2fsprogs.sourceforge.net/e2fsprogs-release.html##1.47.0))
+- multipath-tools ([0.9.4](https://github.com/opensvc/multipath-tools/commits/0.9.4))
```

Comment:
Missing updates?
```
- libarchive ([3.6.2](https://github.com/libarchive/libarchive/releases/tag/v3.6.2))
- xz-utils ([5.4.2](https://github.com/tukaani-project/xz/releases/tag/v5.4.2))
```

Reply:
Oops, I used my local outdated document with the changes listed above. Thanks for spotting, will amend.